stories/Modal.story.jsx

```diff
@@ -27,7 +29,10 @@ storiesOf("Modal", module)
             Launch composed modal
           </Button>
           {open && (
-            <Modal onClose={() => toggleModal(false)}>
+            <Modal
```

Comment:
please add all modal combinations

Reply:
I will add when there is usecase for this. The api is obviously extendable here. 
If you both agree that this is part of this PR even though there is no use-case in our product for that to exist then I will go ahead and add it. But for now I would prefer if we dont add it.
@Weetbix @lanwen your thoughts please?

Reply:
thats quite easy to add and better to do it now, to not switch context next time. Also I think we will get usecase quite soon. In terms of why to add usecases here - since the stories is not only about usecases, but also about testing possible variants.

Reply:
yeah sorry I deleted my comment as I started the implementation already. 
here is the comment that I deleted:
I will add when there is usecase for this. The api is obviously extendable here. 
If you both agree that this is part of this PR even though there is no use-case in our product for that to exist then I will go ahead and add it. But for now I would prefer if we dont add it.
@Weetbix @lanwen your thoughts please?